apps/web/pages/[user]/[type].tsx

```diff
@@ -26,7 +26,7 @@ export default function Type({
   away,
   isBrandingHidden,
   rescheduleUid,
-  org,
+  entity,
```

Comment:
What does entity means in this context?

Reply:
It refers to whether this user event type is being accessed from a team or an org, getting the values from eventData.